app/models/piece.rb

```diff
@@ -42,8 +42,17 @@ def blocked?(x, y)
     end
 
     piece_path.each do |square|
-      return true if game.is_occupied?(square[0], square[1])
+      return true if game.square_occupied?(square[0], square[1])
     end
     false
   end
+
+  def move_to!(x, y) # move and capture
+    game
```

Comment:
If you add a final line in the method of `update_attributes(x_position: x, y_position: y)`, it should go away.  Rubocop is falsely erroring since there isn't something after the `if` yet.